packages/localizations/src/pt-BR.ts

```diff
@@ -827,10 +827,10 @@ export const ptBR: LocalizationResource = {
     createOrganization: {
       formButtonReset: 'Cancelar',
       formButtonSubmit: 'Criar nova organização',
-      formFieldInputPlaceholder__name: undefined,
-      formFieldInputPlaceholder__slug: undefined,
-      formFieldLabel__name: undefined,
-      formFieldLabel__slug: undefined,
+      formFieldInputPlaceholder__name: 'Minha organização',
+      formFieldInputPlaceholder__slug: 'minha-organizacao',
+      formFieldLabel__name: 'Nome',
+      formFieldLabel__slug: 'Slug',
```

Comment:
```suggestion
      formFieldLabel__slug: 'minha-org',
```

Leaving it consistent with the current slug label translation that we have for the `CreateOrganization` component form 🫡